w3-store.md

```diff
@@ -340,6 +402,68 @@ interface UploadAddResponse {
 }
 ```
 
+### `upload/get`
+
+> Get metadata about a upload from a space
+
+The `upload/get` capability can be invoked to get metadata about an upload from a [space](#spaces).
+
+This may be used to check for inclusion, or to get the set of CAR shards associated with a root CID.
+
+The `with` resource URI must be set to the DID of the space to get the upload metadata from.
+
+#### Derivations
+
+`upload/get` can be derived from an [`upload/*`](#upload) or [`*`][ucan-spec-top] capability with a matching `with` resource URI.
+
+#### Caveats
+
+The `root` caveat must contain the root CID of the data item to remove.
+
+| `field`   | `value`                           | `required?` | `context`                                                     |
+| --------- | --------------------------------- | ----------- | ------------------------------------------------------------- |
+| `root`    | data CID string, e.g. `bafy...`   |             | The root CID of the upload to get metadata for                |
```

Comment:
required should be true I think

Reply:
yep,  but i think we need to make it clearer that these tables are referring to "when used as an invocation", as the `root` is not required in the schema, as folks should be allowed to make a delegation that grants you `upload/get` on any root. same for `store.get`. My gut feel is we should make the tables present the ucanto schema rather than the "when used as invocation" form.

Reply:
(yep, as i deliberately differed from the other items in this spec where they have marked an optional field as required)

Reply:
ahhhh got it, I agree re: presenting ucanto schema!!